geodetic_utils/include/geodetic_utils/geodetic_conv.h

```diff
@@ -0,0 +1,208 @@
+#ifndef GEODETIC_CONV_H_
+#define GEODETIC_CONV_H_
+
+#include "math.h"
+#include <Eigen/Dense>
+
+namespace geodetic_conv {
+// Geodetic system parameters
+static constexpr double kSemimajorAxis = 6378137;
+static constexpr double kSemiminorAxis = 6356752.3142;
+static constexpr double kFirstEccentricitySquared = 6.69437999014 * 0.001;
+static constexpr double kSecondEccentricitySquared = 6.73949674228 * 0.001;
+static constexpr double kFlattening = 1 / 298.257223563;
+
+class GeodeticConverter
+{
+ public:
+
+  // Default initialisation/copy constructor and assignment operator are OK.
+
+  ~GeodeticConverter()
+  {
+  }
+
+  void initialiseReference(const double lat, const double lon, const double altitude)
+  {
+    // Save NED origin
+    initial_latitude = deg2Rad(lat);
+    initial_longitude = deg2Rad(lon);
+    initial_height = altitude;
+
+    // Compute ECEF of NED origin
+    geodetic2Ecef(lat, lon, altitude, initial_ecef_x_, initial_ecef_y_, initial_ecef_z_);
+
+    // Compute ECEF to NED and NED to ECEF matrices
+    double phiP = atan2(initial_ecef_z_, sqrt(pow(initial_ecef_x_, 2) + pow(initial_ecef_y_, 2)));
+
+    ecef_to_ned_matrix_ = nRe(phiP, initial_longitude);
+    ned_to_ecef_matrix_ = nRe(initial_latitude, initial_longitude).transpose();
+  }
+
+  void geodetic2Ecef(const double lat, const double lon, const double altitude, double& x,
+                     double& y, double& z)
```

Comment:
Parts of Google Style I'm quite happy to be flexible with, but having output variables as pointers, especially in a library that we want everyone to use, makes everything MUCH clearer to the reader.
https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Function_Parameter_Ordering

Then instead of the call being `geodetic2Ecef(lat, lon, atttiude, x, y, z)` and x y and z being non-obviously modified by the function, it becomes: `geodetic2Ecef(lat, lon, atttiude, &x, &y, &z)` which makes it obvious that x y and z will be modified/are the outputs.

Happy to discuss this if you or @enricgalceran strongly disagree, but this convention has saved me so much trouble in the past.